test/spec/lateralus.model.js

```diff
@@ -1,108 +1,90 @@
-          it('Calls the global change: handler once', function () {
-            assert.equal(count, 1);
-          });
-        });
-
-        describe('toString()', function () {
-          var App = getLateralusApp();
-          var app = new App();
-
-          it('Returns internal name', function () {
-            assert.equal(app.model.toString(), 'lateralus-model');
-          });
-        });
+      _.extend(App.prototype, {
+        lateralusEvents: {
+          'change:prop1': function () {
+            count++;
+            this.model.set('prop2', 'bar');
+          }
+        }
       });
 
-      describe('Mixins', function () {
-        describe('addComponent()', function () {
-          var App = getLateralusApp();
-          var app = new App();
+      new App(document.createElement('div'));
 
-          var component = app.model.addComponent(Lateralus.Component);
+      // NOTE: This test was originally written because of a bug wherein
+      // global change: handlers were called redundantly.  It is important
+      // that count is 1 and not 2:
+      // https://github.com/Jellyvision/lateralus/commit/8975b1d3ce52eaad994a7136789d469f181688a6
+      it('Calls the global change: handler once', function () {
+        assert.equal(count, 1);
+      });
+    });
 
-          it('Adds component to the central Lateralus instance', function () {
-            assert.equal(component, app.components.component0);
-          });
-        });
+    describe('toString()', function () {
+      var App = getLateralusApp();
+      var app = new App();
+
+      it('Returns internal name', function () {
+        assert.equal(app.model.toString(), 'lateralus-model');
+      });
+    });
+  });
+
+  describe('Mixins', function () {
+    describe('addComponent()', function () {
+      var App = getLateralusApp();
+      var app = new App();
+
+      var component = app.model.addComponent(Lateralus.Component);
+
+      it('Adds component to the central Lateralus instance', function () {
+        assert.equal(component, app.components.component0);
+      });
+    });
 
-        describe('emit()', function () {
-          var App = getLateralusApp();
+    describe('emit()', function () {
+      var App = getLateralusApp();
 
-          var testWasCalled = false;
-          App.prototype.lateralusEvents = {
-            test: function () {
-              testWasCalled = true;
-            }
-          };
+      var testWasCalled = false;
+      App.prototype.lateralusEvents = {
+        test: function () {
+          testWasCalled = true;
+        }
+      };
 
-          var app = new App();
-          app.model.emit('test');
+      var app = new App();
+      app.model.emit('test');
 
-          it('Emits events to Lateralus', function () {
-            assert.isTrue(testWasCalled);
-          });
-        });
+      it('Emits events to Lateralus', function () {
```

Comment:
Did you know that the function here can take a `done` parameter, so you _could_ write

```
it('Emits events to Lateralus', function (done) {
  App.prototype.lateralusEvents = {
    test: function () {
      assert(true);
      done();
    }
  }

  var app = new App();
  app.model.emit('test');
}
```